R/output_format.R

```diff
@@ -290,6 +291,10 @@ knitr_options_pdf <- function(fig_width,
 #'   chooses default based on \code{to}). This is typically used to force
 #'   the final output of a latex or beamer conversion to be \code{.tex}
 #'   rather than \code{.pdf}.
+#' @param lua_filters  Character vector of file paths to lua filters to use with
```

Comment:
Nitpicking: for consistency with the naming in Pandoc manual https://pandoc.org/lua-filters.html, I'd capitalize `l`, i.e. `Lua filter`. Similarly, I often use `Pandoc` when referring to the software, and `pandoc` only to refer to the command. Not really a big deal. If you want to make such changes, feel free to push to the master branch directly.

Reply:
Oh ok. Sure, naming things is indeed important. Maybe we can share somewhere an index of all words and names that we want to be consitent ?

Reply:
At the moment, I can't think of any other words besides "Pandoc" and "Lua filter", so perhaps we don't need a checklist for this small problem :)

Reply:
On my list it adds up with TinyTeX, and TeX Live 😄 But yes it is small ! I just wondered if it was the tip of the iceberg 😆

Reply:
The only spelling that bothers me is `TinyTex` (similarly, `LaTex`). I can tolerate any other words!